common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4458,6 +4459,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("120s")
 
+  val DYNAMIC_CONFIG_STORE_FS_PATH: OptionalConfigEntry[String] =
+    buildConf("celeborn.dynamicConfig.store.fs.path")
+      .categories("master", "worker")
+      .version("0.5.0")
+      .doc("The path of dynamic config file for fs store backend.")
```

Comment:
Should we explain that if we didn't set it, will use  `$CELEBORN_CONF/dynamicConfig.yaml`?

Reply:
+1